curvesim/pipelines/utils.py

```diff
@@ -100,3 +102,6 @@ def format_info_str(vol_mult_dict):
     info = [f"{base}/{quote}: {mult}" for (base, quote), mult in vol_mult_dict.items()]
     new_line = "\n    "
     return new_line + new_line.join(info)
+
+
+pool_functions = {CurvePool: pool_vol_mult, CurveMetaPool: metapool_vol_mult}
```

Comment:
This won't handle RAI pools until we make the change to remove the `CurveRaiPool` class.

Reply:
Incidentally, often this is considered a module-level constant and so should be at the top with the other constants.